app/controllers/candidate_interface/decisions_controller.rb

```diff
@@ -8,6 +8,7 @@ class DecisionsController < CandidateInterfaceController
 
     def offer
       @respond_to_offer = CandidateInterface::RespondToOfferForm.new
+      @offer_count = @application_choice.application_form.application_choices.offer.count
```

Comment:
```suggestion
      @offer_count = @application_choice.self_and_siblings.offer.count
```

Reply:
Amazing. I've not come across that before. Thanks!